app/_kong_plugins/ai-proxy-advanced/examples/sdk-dynamic-model-selection.yml

```diff
@@ -0,0 +1,44 @@
+title: 'OpenAI SDK: Chat route with dynamic model selection'
+description: |
+  Configure a chat route that reads the target model from the request path instead of hardcoding it in the configuration.
```

Comment:
```suggestion
description: Configure a chat route that reads the target model from the request path instead of hardcoding it in the configuration.
extended_description: |
  Configure a chat route that reads the target model from the request path instead of hardcoding it in the configuration.
```